library.properties

```
@@ -1,5 +1,5 @@
 name=9 Axes Motion
-version=1.1.0
+version=1.2.0
```

```suggestion
version=1.1.0
```

Versioning should be left to the release manager.

Otherwise we might end up with multiple version bumps from each PR that is merged, then another one from the maintainer at the time of the release, rather than a single version bump at the time of the release as it should be.

---

I did this because trullock suggested it but have now reverted it